data/mods/Magiclysm/Spells/biomancer.json

```diff
@@ -32,5 +32,78 @@
     "spell_class": "BIOMANCER",
     "difficulty": 4,
     "max_level": 1
+  },
+  {
+    "id": "vicious_tentacle",
+    "type": "SPELL",
+    "name": "Vicious Tentacle",
+    "description": "This spell extrudes a long nasty whiplike tentacle of sharp bones and oozing acid from your body, it has a long reach attack and vicious damage.",
+    "valid_targets": [ "self" ],
+    "min_range": 0,
+    "max_range": 0,
+    "min_damage": 1,
+    "max_damage": 1,
+    "effect": "spawn_item",
+    "effect_str": "tentacle_whip",
+    "base_casting_time": 350,
+    "base_energy_cost": 375,
+    "min_duration": 3000,
+    "max_duration": 9000,
+    "duration_increment": 3000,
+    "difficulty": 3,
+    "max_level": 20,
+    "spell_class": "BIOMANCER",
+    "energy_source": "MANA"
+  },
+  {
+    "id": "bio_grotesque",
```

Comment:
this spell will need some body parts defined, in preparation for #30845

Reply:
Ohhh yes. I made those biomancer spells the other day before you had that done.